app/src/scene/camera/camera.cpp

```diff
@@ -80,20 +72,82 @@ void Scene::Camera::handleEvent(Event *event)
 
 void Scene::Camera::keyPressEvent(KeyPressEvent *event)
 {
+    processKey(event->key(), true);
 }
 
 void Scene::Camera::keyReleaseEvent(KeyReleaseEvent *event)
 {
+    processKey(event->key(), false);
 }
 
 void Scene::Camera::mouseMoveEvent(MouseMoveEvent *event)
 {
+    if (_firstMouse)
+    {
+        _lastMousePosX = event->x();
+        _lastMousePosY = event->y();
+        _firstMouse = false;
+    }
+
+    _yaw += (event->x() - _lastMousePosX) * 0.05f;
```

Comment:
please create a variable for the result of this math operation since you are using this data more than once